hackclub.com.yaml

```
@@ -2907,3 +2907,8 @@ takeout:
   - ttl: 600
     type: A
     value: 68.183.252.105
+
+aviquo:
+  - ttl: 600
+    type: CNAME
```

```suggestion
    type: A
```
IPv4 address records should use record type A